assets/js/components/billing/PurchaseCreditModal.jsx

```diff
@@ -403,6 +419,34 @@ class PurchaseCreditModal extends Component {
             </div>
           )}
         </div>
+        <div style={{ padding: "20px 0", width: "75%" }}>
+          <Input
+            name="description"
+            value={this.state.description}
+            onChange={(e) => {
+              this.setState({ description: e.target.value });
+            }}
+            addonBefore={
+              <div
+                style={{
+                  display: "flex",
+                  flexDirection: "row",
+                  alignItems: "center",
+                }}
+              >
+                Memo
+                <Popover
+                  content="For Stripe payments only, this will show on the emailed receipt."
```

Comment:
Should we mention something about VAT?

Reply:
good point, i wanted to leave it broad but i can add VAT as an example of what they could put there